src/main/java/lk/gov/health/phsp/entity/Client.java

```diff
@@ -19,8 +18,8 @@
 @XmlRootElement
 public class Client implements Serializable {
 
-//    @OneToMany(mappedBy = "itemClient", fetch = FetchType.LAZY)
-//    private List<ClientEncounterComponentItem> clientEncounterComponentItems;
+    @OneToMany(mappedBy = "itemClient")

```

Comment:
Need to remove this relationship as it causes severe performance issues

Reply:
Yes this is a mistake of conflict resolution, I will correct that